layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend

```diff
@@ -11,14 +11,10 @@ SRC_URI:append = " \
     file://uEnv.txt_internal \
 "
 
-SSTATE_ALLOW_OVERLAP_FILES += "${DEPLOYDIR}/extra_uEnv.txt"
-SSTATE_ALLOW_OVERLAP_FILES += "${DEPLOYDIR}/uEnv.txt_internal"
+SSTATE_ALLOW_OVERLAP_FILES += "${DEPLOY_DIR_IMAGE}/extra_uEnv.txt"
+SSTATE_ALLOW_OVERLAP_FILES += "${DEPLOY_DIR_IMAGE}/uEnv.txt_internal"
 
 
-SRC_URI:append:beaglebone-pocket = " \
-    file://0001-u-boot-Disable-logging-on-UART0.patch \
```

Comment:
This probably will need to get back in some way.

Reply:
Why do we need this patch ? (I can add it back, since I now have the sources in my devtool workspace)

Reply:
If I understand correctly Balena does not wants u-boot 's output on normal UART.

Reply:
But I'll not apply it until the rest works